sandbox/registration-processor-camel-routes-lost-mz.xml

```diff
@@ -105,10 +105,98 @@
+            message="cmd-validator-->operator-validator lost route ${bodyAs(String)}" />
+        <choice>
+            <when>
+                <simple>${bodyAs(String)} contains '"isValid":true'</simple>
+                <to uri="eventbus:operator-validator-bus-in" />
+            </when>
+			<when>
+                <simple>${bodyAs(String)} contains '"isValid":false' and ${bodyAs(String)} contains '"internalError":false'</simple>
+                <to uri="eventbus:message-sender-bus-in" />
+            </when>
+            <when>
+                <simple>${bodyAs(String)} contains '"internalError":true'</simple>
+                <to uri="eventbus:retry-bus-in" />
+            </when>
+            <otherwise>
+                <to uri="eventbus:error-bus-in" />
+            </otherwise>
+        </choice>
+    </route>
+	<route id="operator-validator-->supervisor-validator lost route">
+        <from uri="eventbus:operator-validator-lost-bus-out" />
+        <log
+            message="operator-validator-->supervisor-validator lost route ${bodyAs(String)}" />
+        <choice>
+            <when>
+                <simple>${bodyAs(String)} contains '"isValid":true'</simple>
+                <to uri="eventbus:supervisor-validator-bus-in" />
+            </when>
+			<when>
+                <simple>${bodyAs(String)} contains '"isValid":false' and ${bodyAs(String)} contains '"internalError":false'</simple>
+                <to uri="eventbus:message-sender-bus-in" />
+            </when>
+            <when>
+                <simple>${bodyAs(String)} contains '"internalError":true'</simple>
+                <to uri="eventbus:retry-bus-in" />
+            </when>
+            <otherwise>
+                <to uri="eventbus:error-bus-in" />
+            </otherwise>
+        </choice>
+    </route>
+	<route id="supervisor-validator-->introducer-validator lost route">
+        <from uri="eventbus:supervisor-validator-lost-bus-out" />
+        <log
+            message="supervisor-validator-->introducer-validator lost route ${bodyAs(String)}" />
+        <choice>
+            <when>
+                <simple>${bodyAs(String)} contains '"isValid":true'</simple>
+                <to uri="eventbus:introducer-validator-bus-in" />
+            </when>
+			<when>
+                <simple>${bodyAs(String)} contains '"isValid":false' and ${bodyAs(String)} contains '"internalError":false'</simple>
+                <to uri="eventbus:message-sender-bus-in" />
+            </when>
+            <when>
+                <simple>${bodyAs(String)} contains '"internalError":true'</simple>
+                <to uri="eventbus:retry-bus-in" />
+            </when>
+            <otherwise>
+                <to uri="eventbus:error-bus-in" />
+            </otherwise>
+        </choice>
+    </route>
+	<route id="introducer-validator-->demo-dedupe new route">
+        <from uri="eventbus:introducer-validator-new-bus-out" />
```

Comment:
this should be last bus out. Also osi to demo do we have a route in old one ?

Reply:
removed introducer to demo-dedupe and used lost bus out...

Reply:
removed introducer to demo-dedupe and used lost bus out...

Reply:
removed introducer to demo-dedupe and used lost bus out...